Rakefile

```diff
@@ -33,6 +33,15 @@ task release_all: [:webpack] do
   end
 end
 
+desc "Makes sure all official locales are complete and clean."
+task :check_locale_completeness do
+  DECIDIM_GEMS.each do |gem_name|
+    Dir.chdir("#{File.dirname(__FILE__)}/decidim-#{gem_name}") do
+      system({ "ENFORCED_LOCALES" => "en,ca,es" }, "rspec spec/i18n_spec.rb")
```

Comment:
Shouldn't we make this configurable? Either via params to the task or using the app configured locales?